webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js

```diff
@@ -419,6 +419,13 @@ export class SakaiRubricCriteria extends RubricsElement {
     .done(() => this.letShareKnow())
     .fail((jqXHR, error, message) => {console.log(error); console.log(message);});
   }
+
+ openEditWithKeyboard(e){
```

Comment:
Can you stick to the formatting in the rest of the file? Space before the opening brace and a blank line to start the function body, unless the function is only one line of code.